### DISCUSSION TITLE
Quick select the whole sentence (very helpful when you need to translate foreign language frequently)

### DISCUSSION THREAD

**Discussion by mingoc** [plateaukao/einkbro#354] [Ideas]
I love this app, it is one of my favorite. I used it to read a lot, and most content I read is not in my native language. Hence I need to tap the word or select a sentence to translate very frequently.  If manually select a sentence it is tedious - tap the word, move to start, move to end, then translate.
So it would be really great, if the app can provide an option to quick select the whole sentence. It will help huge for the user to read foreign language webpages.

**Comment by plateaukao** [comment]
Hi @mingoc 

somewhat make it works. not perfect yet, but useful.
will be released in next version.

https://github.com/user-attachments/assets/baa6431d-c9b1-4e47-975d-16cc7d805beb

**Reply by mingoc** [reply]
Wow, I am very exciting to know this feature will be available soon.  Can't wait to try it. Thanks a lot!
